src/types_.py

```diff
@@ -444,6 +561,24 @@ class IndexContentsListItem(typing.NamedTuple):
     rank: int
     hidden: bool
 
+    @property
+    def table_path(self) -> TablePath:
+        """The table path for the item.
```

Comment:
The description should also mention the sanitization applied to the reference_value

Reply:
Done now